resources/views/gallery/edit.blade.php

```diff
@@ -1,42 +1,104 @@
+                            @foreach ($errors->all() as $error)
+                                <li>{{ $error }}</li>
+                            @endforeach
+                        </ul>
+                    </div>
+                    <br/>
+                @endif
+                <form method="post" id="updateAlbum" action="{{ route('galerij.update', $album->id) }}">
+                    @method('PATCH')
+                    @csrf
+                    <div class="form-group">
+
+                        <span class="requiredStar">*</span><label for="title">Titel:</label>
+                        <input type="text" class="form-control" name="title" value="{{ $album->title }}" id="title"/>
+                    </div>
+
+                    <div class="form-group">
+                        <span class="requiredStar">*</span><label for="description">Omschrijving:</label>
+                        <textarea rows="5" class="form-control" name="description"
+                                  id="description">{{ $album->description }}</textarea>
+                    </div>
+
+                    <div class="form-group">
+                        <span class="requiredStar">*</span><label for="date">Datum:</label>
+                        <input type="date" class="form-control" name="date" value="{{ $album->date }}" id="date"/>
+                    </div>
+                </form>
             </div>
- 
-            <div class="form-group">
-                <span class="requiredStar">*</span><label for="description">Omschrijving:</label>
-                <textarea rows="5" class="form-control" name="description" id="description">{{ $album->description }}</textarea>
+        </div>
+
+        <div class="row mt-3 text-center">
+            <div class="col-md-12">
+                @if (session('success'))
+                    <div class="alert alert-success">
+                        {{ session('success') }}
+                    </div>
+                @endif
             </div>
+        </div>
 
-            <div class="form-group">
-                <span class="requiredStar">*</span><label for="date">Datum:</label>
-                <input type="date" class="form-control" name="date" value="{{ $album->date }}" id="date"/>
+        <div class="row mt-3">
+            @if ($pictures->isEmpty())
+                <p>Geen foto's gevonden in fotoalbum.</p>
+            @else
+                @foreach ($pictures as $picture)
+                    <div class="col-md-4 mb-3">
+                        <div class="image-container">
+                            <img src="{{ asset('images/' . $picture->image) }}"
+                                 alt="Afbeelding {{$picture->id}} uit {{$album->title}}" class="img-fluid w-100 h-100">
+                            <input type='hidden' name='images[]' form='deleteForm' value='{{ $picture->id }}' disabled/>
+                            <button onclick="selectPicture(this)" class='delete-button'></button>
```

Comment:
Hoe toegankelijk is het voor screenreaders om deze knoppen te lezen?

Reply:
ook is een meerdere selectie veld niet handig omdat je niet precies weet wanneer iets verwijderd is, te veel stappen imo